core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -191,6 +192,10 @@ protected boolean supportsEmptyNamespace() {
     return false;
   }
 
+  protected String baseTableLocation() {
```

Comment:
```suggestion
  protected String baseTableLocation(TableIdentifier identifier) {
```
then we can move the string concatenation of the base location/namespace/table from all of the other places into this single place, which would make all calling sites simpler

Reply:
Make sense.